server/cluster-name.cpp

```diff
@@ -2,19 +2,24 @@
 // Copyright (c) 2020 LLC «V Kontakte»
 // Distributed under the GPL v3 License, see LICENSE.notice.txt
 
+#include "server/cluster-name.h"
+
 #include <algorithm>
-#include <cstring>
+
 #include <cctype>
+#include <cinttypes>
+#include <cstdio>
+#include <cstring>
 
-#include "server/cluster-name.h"
+#include "common/crc32.h"
 
 ClusterName::ClusterName() {
   set_cluster_name("default");
 }
 
 const char *ClusterName::set_cluster_name(const char *name) noexcept {
   // reserved for shmem, statsd, socket suffixes and prefixes
-  constexpr size_t reserved = 32;
+  constexpr size_t reserved = 42;
```

Comment:
Maybe it should be looks like
```c++
constexpr size_t reserver = std::strlen(socket_prefix)
``` 
To avoid magic numbers

Reply:
Get rid of magic number and add the necessary logic for it